yarn.lock

```diff
@@ -1684,6 +1684,16 @@
     "@types/qs" "*"
     "@types/serve-static" "*"
 
+"@types/express@4.17.8":
```

Comment:
This is not the most recent version used by other APIs. The API package ought to be using `4.17.9` ... double check the package.json file.